test_harness.py

```diff
@@ -121,6 +130,9 @@ def _gen_parser():
     stratisd_cert_parser.add_argument(
         "--verify-devices", help="Verify /dev/disk/by-id devices", action="store_true"
     )
+    stratisd_cert_parser.add_argument(
```

Comment:
Rather than using a number value, use the ```choices``` parameter to ```add_arguments``` and restrict the options to whatever levels the logging module implements.

Reply:
Since we'll be logging for both stratisd_cert.py and stratis_cli_cert.py best to add this argument to the top-level parser, not just to stratisd's sub-parser.

Reply:
In the main method, before ```namespace.func``` is invoked, configure the logger based on the value of the ```--log-level``` option. Instructions for how to do this are in docs at https://docs.python.org/3/howto/logging.html. Search for string "If you want to set the logging level from a command-line option such as:"